### ISSUE TITLE
adjust height

### ISSUE THREAD

**Issue by trendschau** [leemark/better-simple-slideshow#1]
Nice little script, thank you! Is it possible to adjust the height of the slide for different picture-sizes? Would be enough to return the element after click on the button, so I could get the image hight of the current element and adjust the height of the bss-slide...

**Comment by leemark** [comment]
Sorry for taking so long to comment, it's been a busy summer :) Could you clarify this just a bit "return the element after click on the button"? Do you mean in that the click event handlers just need to return the slideshow element? A pull request or further suggestions would be most welcome!

**Comment by ptrsmk** [comment]
@trendschau might be looking for what I'm wondering, which is...

Is it possible to get the slideshow to nicely display (centered) images when the aspect ratio of the images differ?  As it is, mixing a square image with rectangular images makes the slideshow a square, and rectangular (landscape) images are pushed to the top of the square frame.  One of two alternatives would be better:

1) define a max-height or inherit height from image based on height when width is stretched to 100% of slideshow frame.  Then, center square photo horizontally, leaving empty space to either side.

2) Allow square image to define slideshow frame size, then vertically center landscape photos.

Personally, I prefer the first.  I want the square images that are mixed in to display with even margins on the left and right, while landscape photos display full width and height.

**Comment by cpbl** [comment]
Agreed. Has anyone had luck fiddling with this code so that more than the top of the portrait images in my slideshow are visible (eg in full screen mode)?
